e2e/test/scenarios/embedding/sdk-iframe-embedding-setup/helpers/index.ts

```diff
@@ -26,6 +29,23 @@ export const visitNewEmbedPage = ({ locale }: { locale?: string } = {}) => {
   }
 
   cy.visit("/embed-iframe?" + params);
+
+  if (dismissEmbedTerms) {
```

Comment:
If we've already dismissed the embed terms once (e.g. in user persistence settings where we need to reload the page), then we can't do it again as it won't be there anymore.